src/hearings/store/effects/hearing-actuals.effects.ts

```diff
@@ -30,7 +30,7 @@ export class HearingActualsEffects {
     switchMap((action: any) => this.hearingsService.updateHearingActuals(action.payload.hearingId, action.payload.hearingActuals)
       .pipe(
         tap(() => this.router.navigate([`/hearings/actuals/${action.payload.hearingId}/hearing-actual-add-edit-summary`])),
-        map((response: HearingActualsMainModel) => new hearingActualsActions.UpdateHearingActualsSuccess(response)),
+        map((response: HearingActualsMainModel) => new hearingActualsActions.UpdateHearingActualsSuccess({ ...response, hearingActuals: action.payload.hearingActuals })),
```

Comment:
There is no response data from the api, so just update the action.payload into the store that will be fine.
In theory it should map() first earlier to get the data flush to the store then tap() it will have the data ready to show.

Reply:
The changes are now made after discussing with you and @kuda1992